README.md

```diff
@@ -219,26 +222,27 @@ class JudgementView(views.APIView):
 
 ```
 
-# How to use with function-based views
+When you don't put **keycloak_roles** attribute in the Views that means all methods authorizations will be allowed.
 
-```python
+### Function Based Views
+
+When if you use `api_view` decorator, you would write a very simple `@keycloak_roles` decorator like this:
```

Comment:
It would probably be nice to also write here how to import a decorator? I forgot about it too

Reply:
Cool! well-minded